lib/python/picongpu/plugins/plot_mpl/energy_histogram_visualizer.py

```diff
@@ -44,6 +46,11 @@ def _create_plt_obj(self, idx):
 
         counts, bins, iteration, dt = self.data[idx]
         label = self.sim_labels[idx]
+
+        if np.all(counts == 0.):
+            warn(f"All counts were 0 for {label}. No log-plot can be created!")
```

Comment:
This is not supported by python2
```suggestion
            warn("All counts were 0 for {}. No log-plot can be created!".format(label))
```